adapters/triplelift_native/triplelift_nativetest/supplemental/badlyformed.json

```diff
@@ -1,14 +1,5 @@
 {
-  "expectedMakeRequestsErrors": [
-    {
-      "value": "neither Banner nor Video object specified",
-      "comparison": "literal"
-    },
-    {
-      "value": "No valid impressions for triplelift",
-      "comparison": "literal"
-    }
-  ],
+  "expectedMakeRequestsErrors" : ["no native object specified","No valid impressions for triplelift"],
   "mockBidRequest": {
```

Comment:
Is there a reason why the `badlyformed.json`, `badextbidder.json`, `notgoodstatuscode.json` and `badext.json` tests are no longer needed for the original triplelift adapter tests?

Reply:
no, that was a mistake, and I've restored them.